### PR TITLE
Added solution links to README

### PR DESCRIPTION
**Pull request by aaryan-rampal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,7 +193,13 @@ We already have 2 formatter actions running on every GitHub PR.
 [575. Distribute Candies](https://leetcode.com/problems/distribute-candies/)
 🟢 Easy
 
-[934. Shortest Bridge](https://leetcode.com/problems/shortest-bridge/)
+[733. Flood Fill](https://leetcode.com/problems/flood-fill/)
```
Comment (dataprob) [inline README.md:196]: 733 is repeated twice

File: README.md [hunk 2]
```diff
@@ -193,7 +193,13 @@ We already have 2 formatter actions running on every GitHub PR.
 [575. Distribute Candies](https://leetcode.com/problems/distribute-candies/)
 🟢 Easy
 
-[934. Shortest Bridge](https://leetcode.com/problems/shortest-bridge/)
```
Comment (dataprob) [inline README.md:196]: 934 disappeared

**Review by dataprob — CHANGES_REQUESTED** [review]
In vscode, install the ext, and click the button to see markdown preview.
Screenshot 2023-06-07 at 3 01 49 PM
Screenshot 2023-06-07 at 3 02 08 PM

File: README.md (hunk 2 above)
Comment (aaryan-rampal) [reply README.md:196]: Ah yeah I see what ocurred, meant to delete 733 duplicate

**Comment by dataprob** [comment]
Aaryan @aaryan-rampal also did you update counter at the top?

**Comment by aaryan-rampal** [comment]
> Aaryan @aaryan-rampal also did you update counter at the top?

Nope but fixed it in the commit

**Review by dataprob — APPROVED** [review]
Great work Aaryan @aaryan-rampal !

**Pull request merged**